### PR TITLE
Faster notifications

### PR DESCRIPTION
**Pull request by dzucconi**
Depends on https://github.com/aredotna/api/pull/537

### CODE REVIEW

**Comment by dzucconi** [comment]
This is all quite ugly; I just hacked it in here.

File: components/logged_in_navigation/components/notifications/view.coffee [hunk 1]
```diff
@@ -6,15 +6,22 @@ module.exports = class NotificationsView extends Backbone.View
...
 
-  initialize: ->
+  initialize: ({ @count }) ->
     @listenTo @collection, 'sync', @render
+    @listenTo @count, 'change:unread_count', @render
 
     @$el
-      .one 'mouseenter', => @collection.markRead()
+      .one 'mouseenter', =>
+        @collection.fetch()
```
Comment (broskoski) [inline components/logged_in_navigation/components/notifications/view.coffee]: What does this actually look like? Is it just blank?

Comment (dzucconi) [reply components/logged_in_navigation/components/notifications/view.coffee]: hinotif

Comment (broskoski) [reply components/logged_in_navigation/components/notifications/view.coffee]: Wow, surprising 👍

**Pull request merged**